commands/backups/restore.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+
+function dropboxURL (url) {
+  // Force a dump file to download instead of rendering as HTML
+  // file by specifying the dl=1 param for Dropbox URL
+  if (url.match(/www.dropbox.com/) && !url.endsWith('dl=1')) {
```

Comment:
I fixed this one, which I think pretty much makes the second issue unneeded since you wouldn't be using http auth with dropbox anyways